File: backend/app/schemas/proposicoes.py
from pydantic import BaseModel

class ProposicaoResponse(BaseModel):
    siglaTipo: str
    numero: int
    ano: int
    ementa: str
    link: str | None = None
    proposicao_id: int
    proposicao_uri: str
    #tempo_banco: float = None
    #tempo_api_camara: float = None

class ProposicaoVotosResponse(BaseModel):
    votos_sim: list
    votos_nao: list
    votos_outros: list

File: frontend/src/types/proposicoes.ts
export interface DeputadoVoto {
  deputado_id: number;
  nome: string;
  partido: string;
  uf: string;
}

export interface ProposicaoResponse {
  siglaTipo: string;
  numero: number;
  ano: number;
  ementa: string;
  link: string | null;
  proposicao_id: number;
  proposicao_uri: string;
}

export interface ProposicaoVotosResponse {
  votos_sim: DeputadoVoto[];
  votos_nao: DeputadoVoto[];
  votos_outros: DeputadoVoto[];
}